plugins/image.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: image
+spec:
+  version: v0.1.0
+  homepage: https://github.com/chenjiandongx/kubectl-image
+  shortDescription: Show container images used in the cluster.
+  description: |
+    This plugin shows container images used in the Kubernetes cluster in a
+    table view. You can show all images or show images used in a specified
+    namespace.
+
+    To display a table of all images in current namespace:
+      $ kubectl image
+
+    To display a table of images in dev namespace:
+      $ kubectl image -n dev
+
+    To display a table of images in all namespaces:
+      $ kubectl image --all-namespaces
+  caveats: |
+    Usage:
+      kubectl image
+      kubectl image -n <ns>
+      kubectl image --all-namespaces
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/chenjiandongx/kubectl-image/releases/download/v0.1.0/kubectl-image_darwin_amd64.tar.gz
+    sha256: ef8af1484074f498a078a9b02dee8a5206e9326f4e35dd613a90d480dbc50c1d
+    files:
+    - from: "*"
+      to: "."
```

Comment:
does this copy out a LICENSE file?

at any rate, from:*,to:. is the default, so you can remove these.